src/css/variables.less

```diff
@@ -16,6 +16,9 @@
 @link-color: #ffffff;
 @dropdown-bg-color: #ffffff;
 @dropdown-separator-color: #DADCDE;
+@table-header-color:#46494D;
+@table-highlight-color:#3A3C40;
+@table-border-color:#3C3F43;
```

Comment:
The resulting color is not exactly the same... But we should definitely reduce the number of different colors. May be it's fine for design guys to use a ``ligthen()``-end color... :)